### PR TITLE
Fix formatting EIP-1

### PR DESCRIPTION
**Pull request by fulldecent**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-X.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your Github username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by fulldecent** [comment]
Process: take current code, paste into Typora, copy from Typora and paste back here.

File: EIPS/eip-1.md [hunk 1]
```diff
@@ -5,36 +5,32 @@ status: Active
...
--   A **Meta EIP** describes a process surrounding Ethereum or proposes a change to (or an event in) a process. Process EIPs are like Standards Track EIPs but apply to areas other than the Ethereum protocol itself. They may propose an implementation, but not to Ethereum's codebase; they often require community consensus; unlike Informational EIPs, they are more than recommendations, and users are typically not free to ignore them. Examples include procedures, guidelines, changes to the decision-making process, and changes to the tools or environment used in Ethereum development. Any meta-EIP is also considered a Process EIP.
+- A **Standard Track EIP** describes any change that affects most or all Ethereum implementations, such as a change to the the network protocol, a change in block or transaction validity rules, proposed application standards/conventions, or any change or addition that affects the interoperability of applications using Ethereum. Furthermore Standard EIPs can be broken down into the following categories.
```
Comment (Arachnid) [inline EIPS/eip-1.md]: Why are only some of these bullet points indented?

File: EIPS/eip-1.md [hunk 2]
```diff
@@ -64,88 +60,68 @@ EIPs can also be superseded by a different EIP, rendering the original obsolete.
 
 The possible paths of the status of EIPs are as follows:
 
-![EIP Process](../assets/eip-1/process.png)
+![EIP Process]()
```
Comment (Arachnid) [inline EIPS/eip-1.md]: Why are you deleting the process chart?

File: EIPS/eip-1.md (hunk 1 above)
Comment (fulldecent) [reply EIPS/eip-1.md]: Core, Networking, Interface, ERC are sub categories of Standards Track EIPs. The indentation is subordinated.

Comment (fulldecent) [reply EIPS/eip-1.md]: If you are seeing something else, please you are welcome to make a commit here in this PR.

**Comment by fulldecent** [comment]
Hello team, are there any other remaining comments? I believe all the above reviews are addressed, can this please be merged.

**Pull request merged**